src/NuGet.Clients/PackageManagement.UI/Utility/UIUtility.cs

```diff
@@ -59,7 +59,7 @@ private static bool IsHttpUrl(Uri uri)
 
         // Convert numbers into strings like "1.2K", "33.4M" etc.
         // Precondition: number > 0.
-        public static string NumberToString(long number)
+        public static string NumberToString(long number, IFormatProvider culture = null)
```

Comment:
Instead of the optional parameter this can just require an IFormatProvider and pass current culture from the UI. It would work as it is but we typically try to avoid optional parameters on public methods in this project.